cmd/daemon/flux-notification-controller/server.go

```diff
@@ -0,0 +1,13 @@
+package flux_notification_controller
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func StartHttpServer() {
+	router := mux.NewRouter()
+	router.HandleFunc("/webhook/flux2-alerts", HandleEventFromFlux2).Methods(http.MethodPost)
+	_ = http.ListenAndServe(":3001", router) //TODO: log something
```

Comment:
Let's get an error log in here from the get go 😄 

```suggestion
	err := http.ListenAndServe(":3001", router) //TODO: log something
	if err != nil {
		log.Errorf("Failed to start HTTP server: %v", err)
	}
```